.github/workflows/update-effective-tld-names-file.yml

```diff
@@ -0,0 +1,61 @@
+name: Create a PR if there are changes in effective_tld_names file
+on:
+  schedule:
+    - cron: '0 6 1,15 * *'
```

Comment:
The job will run twice a month on 1 and 15, we can change that.
In any case I have set the workflow_dispatch that allows us to trigger this manually if needed.